packages/lesswrong/lib/vulcan-lib/fragments.ts

```diff
@@ -31,14 +34,19 @@ export const registerFragment = (fragmentTextSource: string): void => {
   // extract subFragments from text
   const matchedSubFragments = fragmentText.match(/\.{3}([_A-Za-z][_0-9A-Za-z]*)/g) || [];
   const subFragments = _.unique(matchedSubFragments.map(f => f.replace('...', '')));
-  
+
+  const sqlFragment = bundleIsServer
```

Comment:
This is kind of a mess, but it gets the job done, and the mess is very localized